tests/lib/rules/no-unknown-property.js

```diff
@@ -46,29 +46,41 @@ ruleTester.run('no-unknown-property', rule, {
     { code: '<div onMouseDown={this._onMouseDown}></div>;' },
     { code: '<a href="someLink" download="foo">Read more</a>' },
     { code: '<area download="foo" />' },
-    { code: '<img src="cat_keyboard.jpeg" alt="A cat sleeping on a keyboard" />' },
+    {
+      code: '<img src="cat_keyboard.jpeg" alt="A cat sleeping on a keyboard" />',
+    },
```

Comment:
i'm not sure what these unrelated formatting changes are - just in case, please be sure to only run prettier on projects that have it configured. it's incorrect to run a formatter on a project that's not part of the project already.